modules/nw-metallb-operator-limit-speaker-to-nodes.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-operator-install.adoc
+
+[id="nw-metallb-operator-limit-speaker-to-nodes_{context}"]
+= Limit speaker pods to specific nodes
+
+By default, when you start MetalLB with the MetalLB Operator, the Operator starts an instance of a `speaker` pod on each node in the cluster.
+You can configure the `MetalLB` custom resource with a node selector to specify which nodes run the `speaker` pods.
```

Comment:
is it appropriate to provide a config example here , in the PR I included a link showing what needs to be done to mark the nodes of interest then use node selector to pick them up ?

Reply:
@msherif1234 , Just below this paragraph, I have an example that deploys the `speaker` pods onto the worker nodes only.

I'm cautious about getting into showing how to add a label on a node because that's covered elsewhere in the doc.  Let me see if I can find a way to link to [Placing pods on specific nodes using node selectors](https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-node-selectors.html#nodes-scheduler-node-selectors).